service/templates/menu.html

```diff
@@ -43,6 +45,7 @@
         </ul>
 
         <ul class="nav navbar-nav navbar-right">
+          <li><a ng-show="showNewUILink" ng-href="[[newUILink]]">Open Page in the New UI</a></li>
```

Comment:
I chose "Open Page in the New UI" because I didn't think "Switch to New UI" made it clear that the user will be redirected to the corresponding page in Spruce by clicking on the link

Reply:
sounds good to me! want to run it by design?